test/features/auth/sign_in_test.rb

```diff
@@ -0,0 +1,32 @@
+require "test_helper"
+
+feature "User Sign In" do
+  scenario "Successful log in to site" do
+    visit root_path
+    click_on "Sign in"
+
+    fill_in "Email", with: users(:user_1).email
+    fill_in "Password", with: (:user_1).password
+    click_on 'Log in'
```

Comment:
Lines 5 through 10 here can also be refactored into the [method described here](https://github.com/trendwithin/thundrs/pull/3/files#r26971333)